site/_docs/avatica_overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: Avatica
-permalink: /docs/avatica.html
+title: Overview
```

Comment:
Make the title "Avatica Overview" otherwise people will think it is an overview of the whole of Calcite.